vendor/src/github.com/docker/notary/tuf/tuf.go

```diff
@@ -248,20 +349,34 @@ func (tr *Repo) InitRoot(consistent bool) error {
 	return nil
 }
 
-// InitTargets initializes an empty targets
-func (tr *Repo) InitTargets() error {
+// InitTargets initializes an empty targets, and returns the new empty target
+func (tr *Repo) InitTargets(role string) (*data.SignedTargets, error) {
+	r := data.Role{Name: role}
+	if !r.IsDelegation() && !(data.CanonicalRole(role) == data.CanonicalTargetsRole) {
```

Comment:
`!(data.CanonicalRole(role) == data.CanonicalTargetsRole)` can be `data.CanonicalRole(role) != data.CanonicalTargetsRole`